contracts/src/CrossChainApplications/ERC20Bridge/tests/ERC20BridgeTests.t.sol

```diff
@@ -637,11 +637,18 @@ contract ERC20BridgeTest is Test {
     }
 
     function testUpdateMinTeleporterVersion() public {
```

Comment:
Curious if you think it'd make sense to remove this test case from `ERC20BridgeTests` given that the implementation is tested in `TeleporterOwnerUpgradeableTests`. I don't think we commonly tests inherited functions in child contract test suites.

Reply:
hmm ye I'd usually agree, but I at least wanted one cross chain app that inherits `TeleporterOwnerUpgradeable` to have a unit test for it, and the other cross chain apps don't have their own unit tests. Also just thought extra unit tests don't hurt.

Reply:
A workaround to this issue would be to implement `TeleporterOwnerUpgradable` in a separate test contract, then move the unit test for `testUpdateMinTeleporterVersion` there.

Reply:
Oh, I see such a contract already exists. I agree with Michael that this unit test would make more sense in `TeleporterUpgradeableTests.t.sol`.

I don't think we sacrifice test coverage in doing so, since `ERC20Bridge` doesn't override this method. If it did, we'd definitely want to test it separately.

Reply:
that's true, since ERC20Bridge doesn't add additional override/functionality and just inherits, having the same test in `TeleporterOwnerUpgradeableTests` is already covering same test coverage. Removed.

Reply:
Thoughts on the same topic but for `Ownable.transferOwnerhip` and `Ownable.renounceOwnership`? I added extra unit tests for those in `TeleporterOwnerUpgradeableTests` though we just inherit the functions in `TeleporterOwnerUpgradeable`. Thinking behind it being these are external contracts, and we can have our own unit tests covering the integration with `updateMinTeleporterVersion`.

Reply:
👍 

Yep that makes sense to me. The unit tests you're talking about that call `transferOwnership` and `renounceOwnership` are actually testing the behavior of `updateMinTeleporterVersion`, given its use of those other methods. Along that line, I would recommend keeping them in place as is, but possibly renaming them to "testCannotUpdateMinVersionAfterOwnershipTransfer", etc (or something along those lines).